src/Orckestra.Composer.Website/App_Data/Razor/Composer/Search/BaseSearchResults.cshtml

```diff
@@ -0,0 +1,195 @@
+                    <img src="/UI.Package/Images/recurring.svg" class="recurring-icon"
+                         alt="@Html.Localize("General", "L_RecurringProduct")"
+                         title="@Html.Localize("General", "L_RecurringProduct")" />
+                </div>
+            </div>
+        </div>
+
+        <template v-if="@(entity).HasPriceRange">
+            <template v-if="@(entity).IsOnSale">
+                <div class="product-tile-special-price">
+                    <small class="text-danger">@Html.Localize("List-Search", "L_PriceFrom")</small>
+                </div>
+                <div class="product-tile-price">
+                    <span class="h4  text-danger">{{@(entity).DisplaySpecialPrice}}</span>
+                </div>
+            </template>
+            <template v-else="v-else">
+                <div class="product-tile-special-price">
+                    <small>@Html.Localize("List-Search", "L_PriceFrom")</small>
+                </div>
+                <div class="product-tile-price">
+                    <span class="h4" data-qa="search-product-price">{{@(entity).DisplayListPrice}}</span>
+                </div>
+            </template>
+        </template>
+        <template v-else="v-else">
+            <template v-if="@(entity).IsOnSale">
+                <div class="product-tile-special-price">
+                    <del>{{@(entity).DisplayListPrice}}</del>
+                </div>
+                <div class="product-tile-price">
+                    <span class="h4  text-danger">{{@(entity).DisplaySpecialPrice}}</span>
+                </div>
+            </template>
+            <template v-else="v-else">
+                <div class="product-tile-special-price">
+                    <!--only used for proper spacing-->
+                </div>
+                <div class="product-tile-price">
+                    <span class="h4" data-qa="search-product-price">{{@(entity).DisplayListPrice}}</span>
+                </div>
+            </template>
+        </template>
+
+        <button class="btn  btn-primary  text-uppercase"
+                data-qa="search-buy-now"
+                v-on:click="addToCart(@entity)"
+                v-bind:disabled="!@(entity).IsAvailableToSell || @(entity).loading">
+            {{@(entity).IsAvailableToSell ? '@Html.Localize("General", "B_AddToCartProductList")' : '@Html.Localize("General", "B_AddToCartProductListOOS")'}}
+            <span v-if="@(entity).loading" class="loading-indicator  fa  fa-spin  fa-fw  fa-circle-o-notch"></span>
+        </button>
+    </div>
+}
+
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://www.composite.net/ns/function/1.0" xmlns:v-bind="v-bind" xmlns:v-on="v-on">
+<head>
+</head>
+<body>
+    <div data-oc-controller="Product.SearchResults"
+         data-context="@JsonConvert.SerializeObject(Results, new Newtonsoft.Json.Converters.StringEnumConverter())"
```

Comment:
As BaseSearchViewModel inherits BaseViewModel =, then you can use here Results.JsonContext -